gateway/1.4/modules/ROOT/pages/_partials/task-config-proxy-protocol.adoc

```diff
@@ -0,0 +1,101 @@
+//tag::intro1[]
+For use cases where Flex Gateway is deployed behind a load balancer or other proxy service, configure Proxy Protocol to preserve client IP addresses across multiple layers of NAT or TCP proxies.
```

Comment:
I thought we only used "where" when referring to location, but the SR isn't definitive.